scripts/publish.js

```diff
@@ -81,6 +81,9 @@ if (/master/.test(currentBranch)) {
 
 // Create a git tag if we are publishing a specific version
 if (!/latest/.test(DOCKER_IMAGE_VERSION)) {
+  shell.exec(`echo ${DOCKER_IMAGE_VERSION} > version.txt`);
+  shell.exec(`git add .`);
```

Comment:
Might be safer to just add the file that you modified. `git add .` might add more and surprise us especially if there are other changes in the folder that were accidentally added.

Reply:
Good point. let me fix that.